content/developers/api-reference/events-api/index.md

```diff
@@ -471,7 +500,7 @@ You should see the response:
 Event records in DataTrails are tokenized at creation time and referred to in all future API calls by a permanent unique identity of the form:
 
 ```bash
-assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+assets/$ASSET_ID/events/$EVENT_ID
```

Comment:
Interestignly this has been changed in the other direction before becasue people complained it was too hard to spot that $ASSET_ID is actually a variable. 
I don't personally see the problem but that's why it looks like that.
I'll leave it to you whether you want to continue with this change but if you do I recommend you also chnage the xxxx.... in the command block above

Reply:
Good catch. I was being surgical and didn't zoom out to see the consistency issue.
I've made them all consistent, including using CAT to generate the file, with a name that can be directly executed.
If someone just wants the json, they can select the mid section, removing the cat/EOF.

Reply:
Might it not be better to use $ASSET_UUID, so as not to be confused with "Asset ID" (which we use elsewhere to include both the `assets/` and the `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`?

Reply:
This has been confusing at times. I'll change to `ASSET_UUID` when referring to just the UUID.

Reply:
done